src/main.h

```diff
@@ -2,6 +2,8 @@
 #define EVE_MAIN_H
 
 #include "controller/controller.h"
+#include "model/world/world.h"
+#include "view/terminalview.h"
```

Comment:
(see main.cpp)
this belongs into controller.h

Reply:
commit 6ff2b4e